Regroup writing guideline sections (#21)
sp-uulm/writing-guidelines: README.md

## General Guidelines

> These guidelines apply to all types of scientific writing at our institute like lecture papers and theses (bachelor and master).

Reviewer: Maybe add "There is also a special section for presentations" - and then in the presentation section say that most things of the other section also hold (although things like "support all your statements..." may be left argumentative as presentations build on top of the writing?

- [Know your dashes!](https://www.merriam-webster.com/grammar/em-dash-en-dash-how-to-use)\
  In LaTeX, we use `-` (-) for hyphens, `--` (&ndash;) for en-dashes (like numeric ranges `pages 12--23`), and `---` for em-dashes &mdash; like this.

### Content and Structure

> - Support all your statements that are not obvious or that you prove by yourself with corresponding citations.

Reviewer: ```suggestion
- Support all your statements.\
  If they are not obvious or that you do not prove claims by yourself, support them with arguments and corresponding citations.
```

Author: For me, the current version is clearer

Reviewer: My main argument is more that this is not restricted to *cites* if you argue for something being better than another thing you shouldn't just leave out the cite if there is none + there shouldn't just be a _cite_ but a reiteration of the argument.

- Include a proper abstract summarizing your work.\

>   Style-wise we recommend you follow a [structured abstract](https://web.archive.org/web/20250807230549/https://procomm.ieee.org/transactions-of-professional-communication/for-prospective-authors/guidelines-to-follow/preparing-structured-abstracts/) approach.

Reviewer: ```suggestion
  We recommend you follow a [structured abstract](https://web.archive.org/web/20250807230549/https://procomm.ieee.org/transactions-of-professional-communication/for-prospective-authors/guidelines-to-follow/preparing-structured-abstracts/) approach.
```

- We do not write click-bait articles.\
  Start with the most important information and the results. List the details afterward. Ideally, your work follows a "pyramid shape" in which you progressively go into more and more detail for the specific aspects of your work.
- The "pyramid" metaphor also holds for the thesis at large. The abstract and the introduction should cover/summarize all parts of the thesis at a high abstraction level. Each individual chapter then goes into the details of that chapter's topic.
- Consequently, already answer your research questions/list your contributions in the introduction.\
  You can also hint at them in the abstract, but your introduction should communicate to the reader what your research questions are and how you answered them (i.e., what your contributions are).
- Plan space for each section before you start writing.\
  Before you start writing, you should plan how you divide the available space for your paper among the sections of your paper.
- Introduce your research questions directly in the introduction.\

>   Although several guidelines recommend to describe the research questions in the methodology, we prefer to directly have them in the introduction to guide the paper.

Reviewer: ```suggestion
  Although some guidelines recommend to describe the research questions in the methodology, we prefer to directly have them in the introduction to guide the paper alongside a short answer. If necessary, you can always reintroduce questions in the methodology.
```

- Use consistent words and terminology to describe your work.\

>   The terminology in your introduction should match the terminology in the research questions, the methodology, and the results.

Reviewer: ```suggestion
  The terminology in your introduction should match the terminology in the research questions, the methodology, and the results. You do not have to avoid duplication &mdash; if you call something `result` do not use `outcome` to refer to the same concept later.
```

- For bigger segments like chapters and sections, include [topic-sentences](https://web.archive.org/web/20250913173407/https://academic-englishuk.com/topic-sentences/).\
  These summarize the content of the respective segment and help the reader to quickly grasp the content (e.g., when skimming through the document).
- Figures and tables should occur before they are referenced in the text.\

>   Especially, figures and tables should not be referenced to the next page (i.e., not on a page after the page of the figure/table reference).

Reviewer: ```suggestion
  Especially, figures and tables should not be referenced to the next page (i.e., not on a page after the page of the figure/table reference). Packages like [placeins](https://ctan.org/pkg/placeins) and [float](https://www.ctan.org/pkg/float) can help you to ensure this automagically, even though we suggest you defer fine-tuning adjustment *after* you have completed the main body of text.
```

- Know your limits, know your threats.\

>   Threats to validity are an important part of scientific work. Make sure to discuss the limitations of your work and how they might threaten the validity of your results (e.g. [Types of Validity](https://web.archive.org/web/20250620015439/https://tagvault.org/blog/types-of-validity/), also consider [Lago et al., ESEM '24](https://doi.org/10.1145/3674805.3686691)).

Reviewer: ```suggestion
  Threats to validity are an important part of scientific work. Make sure to discuss the limitations of your work and how they might threaten the validity of your results (e.g., [tagvault.org: Types of Validity](https://web.archive.org/web/20250620015439/https://tagvault.org/blog/types-of-validity/), also consider [Lago et al., ESEM '24](https://doi.org/10.1145/3674805.3686691)).
```

> 

Reviewer: ```suggestion
- Focus your limitations.\
  When describing your limitations, prioritize your major results (proportionally). If you claim "I am the first to do this correctly", maybe do not focus your limitations on the time performance of your approach!

```

- [Know your dashes!](https://www.merriam-webster.com/grammar/em-dash-en-dash-how-to-use)\
  In LaTeX, we use `-` (-) for hyphens, `--` (&ndash;) for en-dashes (like numeric ranges `pages 12--23`), and `---` for em-dashes &mdash; like this.

### Content and Structure

- Support all your statements that are not obvious or that you prove by yourself with corresponding citations.
- Include a proper abstract summarizing your work.\
  We recommend you follow a [structured abstract](https://web.archive.org/web/20250807230549/https://procomm.ieee.org/transactions-of-professional-communication/for-prospective-authors/guidelines-to-follow/preparing-structured-abstracts/) approach.

> - We do not write click-bait articles.\

Reviewer: Klarer machen was wir hier meinen

Reviewer: reformulate whether gen purpose applicable

Author: I outsourced this to #22
